tests/test_counting_hash.py

```diff
@@ -323,7 +323,10 @@ def test_save_load():
     hi.save(savepath)
 
     ht = khmer.CountingHash(12, sizes)
-    ht.load(savepath)
+    try:
```

Comment:
this try/except should never be triggered unless we're futzing with file formats, right? so... why handle the IOError? Don't we normally expect the tests to pass?